src/components/Menu/index.tsx

```diff
@@ -120,9 +120,12 @@ export function Item({children, label, style, onPress, ...rest}: ItemProps) {
       onFocus={onFocus}
       onBlur={onBlur}
       onPress={async e => {
-        await onPress(e)
-        if (!e.defaultPrevented) {
-          control?.close()
+        if (e.defaultPrevented) {
+          await onPress(e)
+        } else {
+          control?.close(() => {
+            onPress(e)
+          })
```

Comment:
@estrattonbailey would it be okay for us to remove this `e.defaultPrevented()` entirely for native? Are there cases where we want something to run _before_ we close the sheet? I assume the `defaultPrevented()` is only useful on web but maybe I'm wrong.

Reply:
Yeah let's just remove this. I did another pass and I don't see any use cases atm, and even if we want something like this we can achieve it in other ways.